cyipopt/scipy_interface.py

```diff
@@ -185,12 +216,46 @@ def get_bounds(bounds):
         return lb, ub
 
 
+def get_sparse_jacobian_structure(constraints, x0):
```

Comment:
This would be considered a public function. Either add a full docstring and include in the API docs or preprend the name with an underscore to make it private.

Reply:
Done.